cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,24 @@
 
 set -ex
 
+function docker_pull_retry {
+    retry=0
+    maxRetries=5
+    retryInterval=3
```

Comment:
nit: Maybe rename to `retryAfterSeconds` to be clear what this is about.

Reply:
done